ios/Classes/FlutterDownloaderPlugin.m

```diff
@@ -860,7 +860,7 @@ - (void)URLSession:(NSURLSession *)session downloadTask:(NSURLSessionDownloadTas
         NSString *taskId = [self identifierForTask:downloadTask];
         int progress = round(totalBytesWritten * 100 / (double)totalBytesExpectedToWrite);
         NSNumber *lastProgress = _runningTaskById[taskId][KEY_PROGRESS];
-        if (([lastProgress intValue] == 0 || (progress > [lastProgress intValue] + STEP_UPDATE) || progress == 100) && progress != [lastProgress intValue]) {
+        if (([lastProgress intValue] == 0 || (progress > ([lastProgress intValue] + stepUpdate)) || progress == 100) && progress != [lastProgress intValue]) {
```

Comment:
@nimisis seems `stepUpdate` should in fact be `_stepUpdate` without a property.

Reply:
Please correct it to merge.
Thanks

Reply:
Good spot. Thanks. Done.